Megaparsec/Printable.lean

```diff
@@ -98,9 +99,10 @@ instance : Printable UInt8 where
 
 open ByteArray in
 instance : Printable Bit where
-  showTokens
-    | ⟦b⟧ => s!"'{b}'"
-    | nl => let rec go b xs := match xs with
+  showTokens bs :=
+    match bs with
```

Comment:
```suggestion
  showTokens :=
```

Reply:
Same here, I don't understand the suggestion.

![image](https://user-images.githubusercontent.com/66186054/215287784-bb3fd6f4-fda5-4314-9dfa-7d1e7fd1cf9a.png)


Not because the suggestion is bad, but because the UI is very confusing, since it doesn't show the context.

Reply:
It shows number lines and the file, which you can look up in the 'files changed' tab!

I agree context display could be better, but the overall I like the UI for suggestions and will continue to use them, so I'd really appreciate it if you spent a couple of minutes familiarising yourself with it :3

Reply:
I didn't know about "files changed" trick. Thanks!

Without it, the UX is pure garbage imho. I think better UX would be to show the lines and say "there's a suggestion" and a button "view suggestion in files changed".

Anyway, thanks for teaching me how to look at them! Now I'll be less intimidated about reacting to suggested changes.

I assume these aren't tested and you wrote them by hand in Github UI, right?

Reply:
Oh yeah, the suggestions/line-specific comments even show up in that tab!

Yeah, both are hand-written.

Reply:
Yes